docs/source/standard.rst

```diff
@@ -0,0 +1,53 @@
+Card Game Notation Standard
+===========================
+This standard uses TOML_ as notation language.
```

Comment:
what was the reason for toml again?
why not one of the other formats:
yaml, json, xml, edn

Reply:
I believe we thought it was the most human readable language.

Reply:
so our goal is human readability?
I don't think people would want to write this format by hand, it's way too verbose

Reply:
Yes, read only. I don't feel we should invent something ourselves. Any suggestions what else we could use?

Reply:
JSON is ubiquitous, but very verbose.
YAML is bad because it uses semantic whitespace
So I'm fine with TOML

Reply:
Another alternative is to separate the data model from the serialization format.
So we can choose what format to use based on the context/at runtime